OWNERS

```diff
@@ -1,10 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 approvers:
 - brendandburns
-- drubin
+- mstruebing
+- davidgamero
 reviewers:
 - brendandburns
-- drubin
-- itowlson 
+- mstruebing
+- davidgamero
 emeritus_approvers:
 - mbohlool # 10/22/2020
+- drubin # 11/23/2023
```

Comment:
add itowlson here too.

Reply:
I've added him, I wasn't sure as the documentation explicitly states approvers but he was in the reviewers section.